x/btcstaking/keeper/power_dist_change.go

```diff
@@ -113,6 +113,7 @@ func (k Keeper) ProcessAllPowerDistUpdateEvents(
 	events []*types.EventPowerDistUpdate,
 	maxActiveFps uint32,
 ) *types.VotingPowerDistCache {
+	height := uint64(sdk.UnwrapSDKContext(ctx).HeaderInfo().Height)
```

Comment:
```suggestion
	height := uint64(sdk.UnwrapSDKContext(ctx).BlockHeight())
```

Reply:
I remember `BlockHeight` will be deprecated?

Reply:
oh, did not know about this

Reply:
@SebastianElvis is correct https://github.com/cosmos/cosmos-sdk/blob/c4d9a495052b78a02723ed1df1d336efe83a4e8d/types/context.go#L45. `BlockHeight()` would return `header.height` which is deprecated, but not sure if `HeaderService` refers to `headerInfo`

Reply:
> but not sure if HeaderService refers to headerInfo

It is mentioned somewhere in Cosmos SDK code but I forget where :sweat: